motor_control/Triple_HBridge/3PhaseMotorControl_IR2104.ino

```diff
@@ -0,0 +1,158 @@
+#include <ramping.h>
+
+int outHighPins[3] = { 10, 6, 4};
+int shutPins[3] = {9, 5, 3};
+int inPins[3] = {0,1,2};
+int throttlePin = A0;
+bool pin1_state = 0;
+bool pin2_state = 0;
+bool pin3_state = 0;
+int motor_speed = 0;
+int speed_max = 0;
+
+#define RAMP_SPEED 2;
+
+void setup() {
+ //For throttle controls
+// Serial.begin(9600);
+//  pinMode(A0, INPUT);
+ 
+  for(int j = 0; j < 3; j++) {

```

Comment:
Im assuming there will never be more than 3 pins so its ok to hardcode this "3" here